spec/javascript/components/applications/application_form/PaperApplicationForm.test.js

```diff
@@ -9,7 +9,8 @@ import application from '../../../fixtures/domain_application'
 import lendingInstitutions from '../../../fixtures/lending_institutions'
 
 const mockSubmitApplication = jest.fn()
-const applicationWithInvalidAnnualIncome = clone(application)
+
+let testApplication = null
```

Comment:
this refactor is because originally I had tried to add a test to this file, but interactivity of preferences wasn't working right for me 🙁